lib/http-context.js

```diff
@@ -433,19 +433,26 @@ function toJSON(input) {
   }
 }
 
-function toXML(input) {
+function toXML(input, options) {
   var xml;
+  var xmlDefaultOptions = { declaration : true };
+  var xmlOptions = util._extend(xmlDefaultOptions, options);
   if (input && typeof input.toXML === 'function') {
     xml = input.toXML();
   } else {
-    if (input) {
+    if (typeof(input) == 'string') {
+      //  this is a hack to handle js2xmlparser throws syntax error on string
+      //  wraps string from "a string" to "\"a string\""
+      input = JSON.stringify(input);
```

Comment:
@ritch  we could, but we will also need to handle escaping the double quotes in the string itself, not sure any other stuff `JSON.stringify` escapes, I feel JSON.stringify might be easier to maintain, any thoughts?